build_pearl.sh

```diff
@@ -0,0 +1,27 @@
+cd "$(dirname "$0")"
```

Comment:
We need the valory header here. [Example](https://github.com/valory-xyz/trader-quickstart/blob/main/run_service.sh#L3)

Reply:
Added.